bin/metanorma

```diff
@@ -3,19 +3,56 @@
 
 require 'rubygems'
 require 'bundler/setup'
+require 'openssl'
+require 'open-uri'
+require 'net/https'
+require 'tempfile'
+
+def determine_cert_path
+  return 'D:/__enclose_io_memfs__/local/vendor/cacert.pem.mozzila' if Gem.win_platform?
+  path = '/__enclose_io_memfs__/local/vendor/cacert.pem.mozzila'
+  return path if File.file?(path)
+
+  File.join('vendor', 'cacert.pem.mozzila')
+end
+
+def cert_file_path
+  cert_tempfile = Tempfile.new
+  cert_tempfile.tap { |n| n.puts(File.read(determine_cert_path)) }.close
+  cert_file_path = cert_tempfile.path
+end
+
 dl_ext = (RbConfig::CONFIG['host_os'] =~ /darwin/ ? 'bundle' : 'so')
 LIBSASS_TEMP_WRAPPER = "libsass.#{dl_ext}"
+cert_file_path = nil
+DEBUG = ENV['DEBUG']
 
 begin
   # HACK: create temp libsass wrapper in current directory to use with ffi
   require 'bundler/cli'
   require 'bundler/cli/common'
-  sassc_path = Bundler::CLI::Common
-               .select_spec('sassc', :regex_match).full_gem_path
+
+  sassc_path = Bundler::CLI::Common.select_spec('sassc', :regex_match).full_gem_path
   File
     .new(LIBSASS_TEMP_WRAPPER, 'wb')
     .puts(File.read(File.join(sassc_path, 'ext', LIBSASS_TEMP_WRAPPER)))
 
+  # Check ssl availability, if not use vendor ssl certificate
+  begin
+    Net::HTTP.get(URI('https://www.iso.org/'))
+  rescue OpenSSL::SSL::SSLError
+    puts('Canot use ssl requests, instaling custom certificate') if DEBUG
```

Comment:
"Cannot" instead of "Canot". "SSL" instead of "ssl".

Reply:
Oh gosh, that's embarrassing. Will fix in v0.0.8

Reply:
No worries, sometimes a review would help before merging 😉  Thanks @w00lf!